### ISSUE TITLE
Make IRuntimeJobRegistry API less "redundant" when registering a new Job

### ISSUE THREAD

**Issue by nulltoken** [NCronJob-Dev/NCronJob#122]
**Is your feature request related to a problem? Please describe.**
As pointed out in #119, reusing the `NCronJobOptionBuilder` when adding new jobs through `IRuntimeJobRegistry` leads to some stuttering code

```
var registry = provider.GetRequiredService<IRuntimeJobRegistry>();

registry.AddJob(n => n.AddJob<SomeJob>());
```

**Describe the solution you'd like**
This `AddJob(AddJob)` reads a bit weird.

Maybe using an alternative naming would feel less strange from the consumer standpoint?

```
registry.Enrich(n => n.AddJob<SomeJob>());
registry.Register(n => n.AddJob<SomeJob>());
registry.Configure(n => n.AddJob<SomeJob>());
...
```

**Comment by linkdotnet** [comment]
My currently most liked one "Register" or  "RegisterJobs", given that one can add multiple.

**Comment by linkdotnet** [comment]
Tackled and solved in #127
